config/jobs/kubernetes-sigs/kernel-module-management/kernel-module-management-presubmits.yaml

```diff
@@ -12,6 +12,37 @@ presubmits:
       - image: golang:1.18
         command:
         - ci/prow/build
+  - name: pull-kernel-module-management-check-api-changes
+    always_run: true
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-kernel-module-management
+      testgrid-tab-name: check-api-changes
+      description: "make sure deployment manifests and bundle are up-to-date."
+    spec:
+      containers:
+      - image: docker.io/library/golang:1.19-alpine3.16
+        command: [sh]
+        args:
+        - -c
+        - |
+          apk add bash curl gcc git make
```

Comment:
Any reason not to use the test-infra git image?

Reply:
Do they have Go?

Reply:
`make manifests` is running `kubectl` and expects a valid `KUBECONFIG`.
```
ybettan kernel-module-management (main) $ make manifests
/home/ybettan/go/src/github.com/kernel-module-management/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
error: Missing or incomplete configuration info.  Please point to an existing, complete config file:


  1. Via the command-line flag --kubeconfig
  2. Via the KUBECONFIG environment variable
  3. In your home directory as ~/.kube/config

To view or setup config directly use the 'config' command.
make: *** [Makefile:100: manifests] Error 1
```

Reply:
What do you think about using `docker.io/bitnami/git` as the image and removing the installations of:
* `git`
* `bash`
* `curl`
?

Reply:
Does it have Go? It is less expensive to install `git`, `bash` and `curl` than Go.

Reply:
It doesn't. I though you are using a pure alpine image (missed to `go` part in the image name).
Let's merge it then.